zipkin-storage/mysql-v1/src/test/resources/mysql.sql

```diff
@@ -0,0 +1,62 @@
+--
```

Comment:
Sorry for the late answer. I made this answer 3 days ago, but it has not been shown to you because I did not submit it. (weird UI :cry: )

Reply:
@llinder  May I get some feedback for my  answer please?

Reply:
Can't we use the one in `src/main/resources/mysql.sql` with a default value for $row_format_option or keep it as it is and set `ROW_FORMAT=COMPRESSED` globally when needed?

Reply:
Personally I would really like to avoid the duplicate file. If we can add a variable with a default as @jcchavezs suggested that would be ideal. I can try to give it some more thought when I have some free time but right now I don't have other suggestions.

Reply:
Thanks for the feedback. I will give it a try that way. (not sure if I could make it this year) Happy Christmas season for both of you! :wink:

Reply:
I've made a commit to get around the issue. It leaves the file `mysql.sql` intact and removes the option `ROW_FORMAT=COMPRESSED` from the copied twin during the docker build for `s390x`. I hope that the change would meet your expectations. I look forward to your opinions about this.